tests/arp/test_arp_dualtor.py

```diff
@@ -10,7 +10,9 @@
 from ipaddress import ip_network, IPv6Network, IPv4Network
 from tests.arp.arp_utils import clear_dut_arp_cache, increment_ipv6_addr, increment_ipv4_addr
 from tests.common.helpers.assertions import pytest_assert, pytest_require
-from tests.common.fixtures.ptfhost_utils import change_mac_addresses
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses                                # lgtm[py/unused-import]
+from tests.common.storage_backend.backend_utils import skip_test_module_over_backend_topologies     # lgtm[py/unused-import]
+
 
 pytestmark = [
     pytest.mark.topology('t0', 'dualtor')
```

Comment:
We should not skip the tests in this file since they are applicable to regular T0 topology as well